tests/php/Service/BreakoutRoomServiceTest.php

```diff
@@ -51,6 +51,8 @@ class BreakoutRoomServiceTest extends TestCase {
 	private $participantService;
 	/** @var ChatManager|MockObject */
 	private $chatManager;
+	/** @var INotificationManager|MockObject */
+	private $notificationManager;
```

Comment:
**Optional**:
Union types works native since PHP 8.0 (https://3v4l.org/ArLAg):
```suggestion
	private INotificationManager|MockObject $notificationManager;
```
Maybe we can follow changing the code that we touch removing the union type from annotation and use the native way.

Reply:
Moved to https://github.com/nextcloud/spreed/issues/8677